courses/machine_learning/deepdive2/structured/labs/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>this is different from the solution notebook?</p><p>no IMAGE for example...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Updated it thanks!</p>